packages/subgraph/schema.graphql

```diff
@@ -107,70 +107,102 @@ type Dao @entity {
   balances: [Balance!] @derivedFrom(field: "dao")
   contractPermissionIds: [ContractPermissionId!]! @derivedFrom(field: "dao")
   permissions: [Permission!]! @derivedFrom(field: "dao")
-  plugins: [DaoPlugin!] @derivedFrom(field: "dao")
+  plugins: [PluginInstallation!]! @derivedFrom(field: "dao")
```

Comment:
if i querry this is there a way to know what plugin is intalled ?
is it multisig or tokenVoting...etc?

Reply:
I moved now the installation to `pluginInstallations` and left `plugins` to reflect with `IPlugin`